src/routes/open/components/Layout.tsx

```diff
@@ -139,5 +143,3 @@ const Layout = (props: LayoutProps): React.ReactElement => {
     </>
   )
 }
-
-export default Layout
```

Comment:
is there a reason to remove default?

Reply:
Yep, check this: https://basarat.gitbook.io/typescript/main-1/defaultisbad

Reply:
Interesting, I disagreed with some points and some others are interesting.  
But I think we should all agreed on this because right now almost all the project is making use of default export. 

If we define to start using named exports always, we should define a linter rule. wdyt?

Reply:
Yep it's a good idea for me